modules/dynamic/smithy/dynamic.smithy

```diff
@@ -184,11 +184,20 @@ structure ServiceShape {
   version: String,
   errors: MemberList,
   operations: MemberList,
+  resources: MemberList,
   traits: TraitMap
 }
 
-/// TODO
 structure ResourceShape {
+  /// ignored: identifiers, properties, collectionOperations
```

Comment:
Couldn't get `collectionOperations` to work. Same applied for `list` member shape. Looks like they have some specific requirements, as I get following error: 
```
smithy4s.example#ListPublishers: This operation is bound as a collection operation on the `smithy4s.example#Publisher` resource, but it improperly binds all of the identifiers of the resource to members of the operation input. 
```
Will try to solve that and push a fix for it

Reply:
guess we can follow up with that